### DISCUSSION TITLE
Authorization request failure

### DISCUSSION THREAD

**Discussion by rprakhya** [Genuitec/Copilot4Eclipse-community#62] [Help] [answered]
getting an Authorization request failure on my eclipse while i see "Your device is connected" on my github

https://github.com/login/device/success

Any idea why it is not able to Authorize?

See this : Authorization request failed: your subscription may be expired. Please try again to resume use of GitHub copilot features

**Comment by rprakhya** [comment]
![image](https://github.com/user-attachments/assets/3ba4b5b6-dd86-4c2c-92c7-c3541e52ad5a)

**Reply by wayneparrott** [reply]
Before digging in deep please ensure that your Copilot Subscription is valid. 

Open the GitHub dashboard at [Your Copilot](https://github.com/settings/copilot) and verify that you have a valid active Copilot subscriptions similar to the following screenshot. Let us know what you determine.
![image](https://github.com/user-attachments/assets/59855985-21e1-4939-96e2-ed23edcf7fec)

**Reply by Nivanchenko** [reply]
does it work with new free version of copilot?

**Reply by wayneparrott** [reply]
@Nivanchenko yes but YMMV. 
Presently I am able to use Copilot4Eclipse with the new free individual GitHub Copilot access. We have no way of determining how long this access will be available. It could randomly change in the future. But between now and then, enjoy.

**Comment by rprakhya** [comment, marked as answer]
This issue is resolved for me. Enterprise access has granted to me and i'm good. thanks for your feedback so far on this.

**Reply by wayneparrott** [reply]
Thanks for the update. Best wishes!
